src/ast/passes/codegen_llvm.cpp

```diff
@@ -3698,7 +3718,7 @@ void CodegenLLVM::generate_maps(const RequiredResources &required_resources,
     createMapDefinition(stack_type.name(),
                         libbpf::BPF_MAP_TYPE_LRU_HASH,
                         128 << 10,
-                        MapKey({ CreateUInt64(), CreateUInt32() }),
+                        CreateArray(12, CreateInt8()),
```

Comment:
Nit: would something like this work to make it more descriptive?
```suggestion
                        CreateTuple({ CreateUInt64(), CreateUInt32() }),
```

Reply:
I tried that at first but there was an issue with resolving the stack id. Let me work on fixing this in a follow-up PR.